presto-main/src/main/java/com/facebook/presto/sessionpropertyproviders/NativeWorkerSessionPropertyProvider.java

```diff
@@ -0,0 +1,216 @@
+                                "If set to zero, buffering is disabled.",
+                        1024L * 1024L,
+                        !nativeExecution),
+                stringProperty(
+                        NATIVE_SPILL_FILE_CREATE_CONFIG,
+                        "Native Execution only. Config used to create spill files. This config is \n" +
+                                "provided to underlying file system and the config is free form. The form should be\n" +
+                                "defined by the underlying file system.",
+                        "",
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_JOIN_SPILL_ENABLED,
+                        "Native Execution only. Enable join spilling on native engine",
+                        false,
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_WINDOW_SPILL_ENABLED,
+                        "Native Execution only. Enable window spilling on native engine",
+                        false,
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_WRITER_SPILL_ENABLED,
+                        "Native Execution only. Enable writer spilling on native engine",
+                        false,
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_ROW_NUMBER_SPILL_ENABLED,
+                        "Native Execution only. Enable row number spilling on native engine",
+                        false,
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_TOPN_ROW_NUMBER_SPILL_ENABLED,
+                        "Native Execution only. Enable topN row number spilling on native engine",
+                        false,
+                        !nativeExecution),
+                integerProperty(
+                        NATIVE_SPILLER_NUM_PARTITION_BITS,
+                        "Native Execution only. The number of bits (N) used to calculate the " +
+                                "spilling partition number for hash join and RowNumber: 2 ^ N",
+                        3,
+                        !nativeExecution),
+                booleanProperty(
+                        NATIVE_DEBUG_VALIDATE_OUTPUT_FROM_OPERATORS,
+                        "If set to true, then during execution of tasks, the output vectors of " +
+                                "every operator are validated for consistency. This is an expensive check " +
+                                "so should only be used for debugging. It can help debug issues where " +
+                                "malformed vector cause failures or crashes by helping identify which " +
+                                "operator is generating them.",
+                        false,
```

Comment:
the native_debug... fields should be hidden always (that was their default before)